### PR TITLE
Remove MCAssoc 🔥 

### PR DESCRIPTION
**Pull request by tadhgboyle**
Microsoft removed the ability to apply a skin automatically via a URL, so MCAssoc has become much more annoying to use for the end user.
Additionally our implementation of it has been broken for a few years, but only a handful of people have ever mentioned it.
The Nameless Plugin is a better option for handling account verification!

### CODE REVIEW

File: core/classes/Core/Util.php [hunk 1]
```diff
@@ -377,6 +377,8 @@ public static function setSetting(string $setting, ?string $new_value, string $m
         } else if ($new_value !== null) {
             $cache[$setting] = $new_value;
         }
+
+        self::setSettingsCache($module, $cache);
```
Comment (Aberdeener) [inline core/classes/Core/Util.php:381]: Important line here, not worth it's own PR

File: modules/Core/pages/register.php [hunk 2]
```diff
@@ -245,152 +210,123 @@
...
+                    }
 
-                        // Hash password first
-                        $password = password_hash($_POST['password'], PASSWORD_BCRYPT, ['cost' => 13]);
-                        $_SESSION['password'] = $password;
-                        unset($_POST['password']);
+                    $password = password_hash(Input::get('password'), PASSWORD_BCRYPT, ['cost' => 13]);
+                    // Get current unix time
+                    $date = new DateTime();
+                    $date = $date->getTimestamp();
```
Comment (supercrafter100) [inline modules/Core/pages/register.php]:
```suggestion
                    $date = date('U');
```

**Review by partydragen — DISMISSED** [review]

**Review by supercrafter100 — CHANGES_REQUESTED** [review]

File: modules/Core/pages/register.php (hunk 2 above)
Comment (supercrafter100) [reply modules/Core/pages/register.php]: I guess you could technically just move this definition straight to the `$user->create()` part

Comment (tadhgboyle) [reply modules/Core/pages/register.php]: Done

**Review by supercrafter100 — DISMISSED** [review]

**Review by partydragen — DISMISSED** [review]

**Review by supercrafter100 — APPROVED** [review]

**Review by partydragen — APPROVED** [review]

**Pull request merged**